src/main/resources/assets/computercraft/lua/rom/autorun/10_require.lua

```diff
@@ -0,0 +1,124 @@
+-- -*- coding: utf8 -*-
+-- Copyright (c) 2014 Odd Straaboe <oddstr13 at openshell dot no>
+-- License: MIT - http://opensource.org/licenses/MIT
+-- Filename: require.lua
+
+_G.package = {}
+
+_G.package.cpath = ""
+_G.package.loaded = {}
+_G.package.loadlib = function() error("NotImplemented: package.loadlib") end
+_G.package.path = table.concat({
+    "?",
+    "?.lua",
+    "?/init.lua",
+    "/lib/?",
+    "/lib/?.lua",
+    "/lib/?/init.lua",
+    "/rom/apis/?",
+    "/rom/apis/?.lua",
+    "/rom/apis/?/init.lua",
+    "/rom/apis/turtle/?",
+    "/rom/apis/turtle/?.lua",
+    "/rom/apis/turtle/?/init.lua",
+    "/rom/apis/command/?",
+    "/rom/apis/command/?.lua",
+    "/rom/apis/command/?/init.lua",
+}, ";")
+_G.package.preload = {}
+_G.package.seeall = function(module) error("NotImplemented: package.seeall") end
+_G.module = function(m) error("NotImplemented: module") end
+
+local _package_path_loader = function(name)
+    
+    local fname = name:gsub("%.", "/")
+    
+    for pattern in package.path:gmatch("[^;]+") do
+        
+        local fpath = pattern:gsub("%?", fname)
+        
+        if fs.exists(fpath) and not fs.isDir(fpath) then
+            
+            local apienv = {}
+            setmetatable(apienv, {__index = _G})
+            
+            local apifunc, err = loadfile(fpath)
+            local ok
+            
+            if apifunc then
+                setfenv(apifunc, apienv)
+                ok, err = pcall(apifunc)
```

Comment:
This could have been done for consistency (`os.loadAPI` will catch errors are merely return `false`) but I agree, it would be better to allow the program to handle errors.